app/styles/layout.scss

```diff
@@ -51,7 +73,56 @@ md-toolbar .md-button {
 }
 
 md-input-container {
-  width: 100%
+  width: 100%;
+}
+
+.view-engineers-container {
+  width: 100%;
+  height: 600px;
+  margin: auto;
+  overflow: scroll;
+
+}
+
+.view-engineers-container md-card {
+  margin: 1px;
+}
+
+//  {
+//   color: black;
+//   border: 2px solid rgba(55, 71, 79, 0.36);
+// }
+
+.text-center .md-button, .view-engineers-container .md-button, form .md-button  {
+  color: black;
+  border: 2px solid rgba(55, 71, 79, 0.36);
+}
+
+.view-engineers-title-container {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 75px;
+}
+
+
+.view-engineers-title-container, .message-container {
+  border-bottom: 5px solid #3f51b5;
+}
+
+.view-engineers-title-container h2 {
+  margin-left: 6px;
+}
```

Comment:
Sass allows for nesting. Lets use that.
```sass
.view-engineers-title-container {
  width: 100%;
  display: flex;
  flex-direction: row;
  align-items: center;
  height: 75px;
  
  h2 {
    margin-left: 6px;
  }
}
```
then you an combine this with declaration on line 101